pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiPicklistOption.java

```diff
@@ -45,6 +46,20 @@ long insert(@Bind("picklistQuestionId") long picklistQuestionId,
                 @Bind("revisionId") long revisionId,
                 @Bind("nestedOptionsTemplateId") Long nestedOptionsTemplateId);
 
+    @UseStringTemplateSqlLocator
+    @SqlBatch("insertOption")
+    @GetGeneratedKeys
+    long[] insert(@Bind("picklistQuestionId") long picklistQuestionId,
+                @Bind("stableId") Iterator<String> stableIds,
+                @Bind("optionLabelTemplateId") Iterator<Long> optionLabelTemplateIds,
+                @Bind("tooltipTemplateId") Iterator<Long> tooltipTemplateIds,
+                @Bind("detailLabelTemplateId") Iterator<Long> detailLabelTemplateIds,
+                @Bind("allowDetails") Iterator<Boolean> allowDetails,
+                @Bind("isExclusive") Iterator<Boolean> isExclusive,
+                @Bind("displayOrder") Iterator<Integer> displayOrder,
+                @Bind("revisionId") long revisionId,
+                @Bind("nestedOptionsTemplateId") Iterator<Long> nestedOptionsTemplateId);
+
```

Comment:
These type of JDBI entries are what empowers the batch inserts. Obviously the trick is that the iterator are called by JDBI for each row inserted, so the iterator streams need to be in sync so they return the correct corresponding values.